stage0/app.go

```diff
@@ -357,7 +357,12 @@ func RmApp(dir string, uuid *types.UUID, usesOverlay bool, appName *types.ACName
 		}
 
 		if err := callEntrypoint(dir, appStopEntrypoint, args); err != nil {
-			return err
+			status, err := common.GetExitStatus(err)
+			// ignore nonexistent units failing to stop. Exit status 5
+			// comes from systemctl and means the unit doesn't exist
+			if err != nil || status != 5 {
```

Comment:
in https://github.com/coreos/rkt/pull/3230/files#diff-b5ccff3f3777d1b87c84277733a8c04fR445 you explicitely return `return status.ExitStatus(), nil`, should it rather be `return status.ExitStatus(), err`?